samples/sid_end_device/prj.conf

```diff
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
+CONFIG_STATIC_INIT_GNU=y
```

Comment:
build test failed (github actions):
samples/sid_end_device/prj.conf:6: warning: attempt to assign the value 'y' to the undefined symbol STATIC_INIT_GNU